packages/create-crc-app/templates/README.md

```diff
@@ -36,8 +40,10 @@ Update `config/dev.webpack.config.js` according to your application URL. [Read m
 
 - The starter repo uses Travis to deploy the webpack build to another Github repo defined in `.travis.yml`
   - That Github repo has the following branches:
-    - `stage-beta` (deployed by pushing to `master` or `main` on this repo)
-    - `stage-stable` (deployed by pushing to `master/main-stable` on this repo)
+    - `ci-beta` (deployed by pushing to `master` or `main` on this repo)
+    - `ci-stable` (deployed by pushing to `ci-stable` on this repo)
+    - `qa-beta` (deployed by pushing to `qa-beta` on this repo)
+    - `qa-stable` (deployed by pushing to `qa-stable` on this repo)
```

Comment:
I don't think this change is right

Reply:
I've removed this section as it doesn't apply to what's currently being generated by this package. I've also trimmed down the README to relevant (to this package) information.